### ISSUE TITLE
form.file() with the placeholder option results in invalid placeholder attribute

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1075]
The `placeholder` attribute is invalid on file inputs and should be excluded

```
<input id="file-1710731869" name="inputTextLong" placeholder="Placeholder" type="file" aria-describedby="guid-411430282"  class="form__control file" />
```

**Comment by jamesjacobs** [comment]
Fixed by #1089
